bench/wtperf/wtperf_run_py/validate_expected_stats.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import argparse
+import json
+import os
+
+
+def main():
+    """
+    Validate the expected output from the last run of wtperf_run.py
+    Take in a list of stats and their values as json and verify them against the contents of evergreen_out.json.
+    Example usage:
+        ./validate_perf_stats.py '{"Pages seen by eviction": 200, "Insert count": 1153326}' './evergreen_out.json'
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('expected_stats', type=str, help='The expected stats and their values')
+    parser.add_argument('stat_file', type=str, help='The evergreen stat file produced by the perf test')
+    args = parser.parse_args()
+
+    if os.path.basename(args.stat_file) != 'evergreen_out.json':
```

Comment:
I may in the future(as part of migrating perf-long to evergreen [WT-5014](https://jira.mongodb.org/secure/RapidBoard.jspa?rapidView=467&selectedIssue=WT-7140&projectKey=WT)) append the test name to the evergreen file name (`evergreen_out_${perf_test_name}.json`). Therefore, I was wondering if we can get rid of this conditional statement?

Reply:
Yep I'll modify this to an `evergreen.*.json` regex to support the new naming format.
I want to keep this so people can't pass in atlas_out.json accidentally which will break the content parsing.

Reply:
Updated in b1456fe